src/source-pcap-file-helper.c

```diff
@@ -208,7 +208,7 @@ TmEcode InitPcapFile(PcapFileFileVars *pfv)
         SCReturnInt(TM_ECODE_FAILED);
     }
 
-#if defined(HAVE_SETVBUF) && !defined(OS_WIN32)
+#if defined(HAVE_SETVBUF) && defined(OS_LINUX)
```

Comment:
Strange `setvbuf` returns ok, but the behavior is bad afterwards ?

Reply:
Seems so.